packages/core/src/config/config.ts

```diff
@@ -623,3 +624,12 @@ function assignExisting<T>(target: Record<string, T>, obj: Record<string, T>) {
     }
   }
 }
+
+export function makeConfig(rules: Record<string, RuleConfig>, decorators?:  Record<string, DecoratorConfig>) {
```

Comment:
Keep this function in __tests__ helpers. We don't need this function anywhere in non-test code so why have it here.

Reply:
@RomanHotsiy moved to `core/__test__/utils`